rpm/spec.in

```diff
@@ -76,7 +76,7 @@ mkdir -p $RPM_BUILD_ROOT/
 %{_libdir}/libwolfssl.la
 %{_libdir}/libwolfssl.so
 %{_libdir}/libwolfssl.so.35
-%{_libdir}/libwolfssl.so.35.0.0
+%{_libdir}/libwolfssl.so.35.1.0
```

Comment:
Is this right? Should it be 36.0.1? I don't know much about this file, so just checking.

Reply:
Sure no worries, I think it's right because AGE was incremented in this case. That is the value that a build on Ubuntu gives the resulting libraries and what "make rpm" is looking for when ran.